### PR TITLE
Fix Display error for Duration

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Fixes #259

File: src/statistics.rs [hunk 1]
```diff
@@ -60,7 +60,7 @@ impl Statistics {
         for typ in SWType::iter() {
             if let Some(sw) = self.stopwatches.get(typ) {
                 let sw = sw.borrow();
-                let elapsed = sw.elapsed();
+                let elapsed = sw.elapsed().as_secs();
```
Comment (EPashkin) [inline src/statistics.rs]: IMHO better completely remove this column, only mssec is enough

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by EPashkin** [comment]
:+1:

**Comment by GuillaumeGomez** [comment]
I let you handle the merge then. :wink:

**Comment by gkoz** [comment]
So you've found one more reason to not run `cargo update` randomly? ;)
I'd hold off modifying any `Gir.toml`s for now, because I'm not convinced `[table] # comment` actually is invalid.

**Comment by GuillaumeGomez** [comment]
> So you've found one more reason to not run cargo update randomly? ;)

:-°

**Comment by GuillaumeGomez** [comment]
I actually did because of the compiler error I got. Not a wisely choice... I'm removing the Cargo.lock change.

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by EPashkin** [comment]
IMHO without `cargo.lock` this PR has no meaning

**Comment by GuillaumeGomez** [comment]
:cry:

**Comment by GuillaumeGomez** [comment]
So what do I do?

**Comment by EPashkin** [comment]
Removing `Display` was introduced yesterday in https://github.com/ellisonch/rust-stopwatch/commit/1ffee80fc41a70836e85a8f4f614510c831d78be we just was "lucky" and find it immediately.

**Comment by EPashkin** [comment]
Toml changed month ago https://github.com/alexcrichton/toml-rs/commit/848227c6053019918b726fa7356f5dfb87861420

**Comment by gkoz** [comment]
The toml change was merged only 3 days ago. As to statistics, we ever need to touch that code again, I'd be in favor of just using the `hprof` crate.

**Comment by EPashkin** [comment]
About toml, IMHO in https://github.com/alexcrichton/toml-rs/commit/2cd1116e062f9af389f97044360a9cfbdb127c82 is good reason to restriction

**Comment by EPashkin** [comment]
As we already merged toml-related fix we better forget about it, adding Issue or PR to toml don't change that fact.
So currently we has only these options:
1) Postpone solution,
2) Do minor fix to work with stopwatch 0.0.7 
3) Replace stopwatch with hprof (gkoz suggested)
@GuillaumeGomez If you will do 3th in this PR it be best, but 1) and 2) also good for me

**Comment by EPashkin** [comment]
*both 2 and 3 options must include `Cargo.lock` changes.

**Comment by GuillaumeGomez** [comment]
Please choose and I'll will update in this way.

**Comment by EPashkin** [comment]
As I understand @gkoz stated that him don't like 2 and prefer 3.
I also stated my option.
So final choice up to you, @GuillaumeGomez, if you have time to apply `hprof` then go 3, else close this PR (1).

**Comment by GuillaumeGomez** [comment]
I'll take option 3 then. Sorry @EPashkin :wink:

**Comment by GuillaumeGomez** [comment]
Updated.

File: src/main.rs [hunk 2]
```diff
@@ -63,10 +62,11 @@ fn do_main() -> Result<(), Box<Error>> {
         Err(err) => return Err(Box::new(err)),
     };
 
-    let mut statistics = Statistics::new();
+    let statistics = Profiler::new("gir profiler");
```
Comment (EPashkin) [inline src/main.rs]: Just "Gir"?

File: src/config/members.rs [hunk 3]
```diff
@@ -1,5 +1,5 @@
 use super::ident::Ident;
-use super::identables::{Identables, Parse};
+use super::identables::Parse;
```
Comment (EPashkin) [inline src/config/members.rs]: Thank for notice that.

File: src/main.rs (hunk 2 above)
Comment (GuillaumeGomez) [inline src/main.rs]: Sure? It isn't very beautiful. I agree for another but just "Gir"? :-/

**Comment by EPashkin** [comment]
Maybe need call `statistics.disable();` after loading config if it disabled.

**Comment by EPashkin** [comment]
* or just don't do `start_frame`

File: src/main.rs (hunk 2 above)
Comment (EPashkin) [inline src/main.rs]: "Timing information for gir profiler:" IMHO no good,
prefer "Timing information for Gir:"

Comment (GuillaumeGomez) [inline src/main.rs]: Fine by me.

**Comment by EPashkin** [comment]
May be better remove `watcher_total`

**Comment by GuillaumeGomez** [comment]
> May be better remove watcher_total

It's "convenient": you don't have to make the addition yourself. :D

File: src/main.rs (hunk 2 above)
Comment (EPashkin) [inline src/main.rs]: IMHO it `tautology`

**Comment by GuillaumeGomez** [comment]
> Maybe need call `statistics.disable();` after loading config if it disabled.

It doesn't impact performance so why bother?

**Comment by GuillaumeGomez** [comment]
So I changed the message. Anything else?

**Comment by EPashkin** [comment]
Thanks, others was just possibilities.

**Comment by EPashkin** [comment]
I leave final decision to @gkoz

**Comment by EPashkin** [comment]
Strange `cargo update` still changes `Cargo.lock`.

**Comment by EPashkin** [comment]
Sorry, I though that you update it.

**Comment by GuillaumeGomez** [comment]
I thought I did as well. Sorry, updating.

**Comment by GuillaumeGomez** [comment]
Ok, now it has been updated.

**Comment by EPashkin** [comment]
:+1:

**Comment by gkoz** [comment]
Looks good. I guess we need to bump rust version in appveyor.

**Comment by GuillaumeGomez** [comment]
Done as well.

**Comment by gkoz** [comment]
Thanks

**Pull request merged**